assets/scss/base.scss

```diff
@@ -362,72 +353,11 @@ a.v-btn {
 // Common button end
 
 .h-divider {
-  margin: 0;
-  border-color: #e2e8ee;
+  border-width: 1px;
   border-style: solid;
-  border-width: 3px;
-  box-shadow: 0 1px 1px $gray4, 0 -1px 1px $gray4;
+  border-color: $gray2-5;
 }
 
-
-@media only screen and (max-width: 640px) {
```

Comment:
I moved these to "layout.scss".

Reply:
Correction - I moved these to "pages/index.vue" (where hero-banner exists; this does not need to be in a global style file as it's only used in 1 component).